airbyte_cdk/sources/declarative/incremental/per_partition_cursor.py

```diff
@@ -325,8 +325,10 @@ def is_greater_than_or_equal(self, first: Record, second: Record) -> bool:
     @staticmethod
     def _convert_record_to_cursor_record(record: Record) -> Record:
         return Record(
```

Comment:
Small not to @tolik0 that this change might impact you (hopefully very slightly) for what you are working on